Makefile

```diff
@@ -10,7 +10,10 @@ black:
 flake8:
 	@poetry run flake8 --ignore=E501,W501,E231,W503
 
-check: black flake8
+mypy:
+	@poetry run mypy scanapi
+
+check: black flake8 # mypy
```

Comment:
@Stannislav hey! 👋🏾  I think we can revert one commit back where `mypy` is no commented out. 

Once https://github.com/scanapi/scanapi/issues/457 is resolved. We can merge this PR just after that. What do you think ?

Also, you might need to add a changelog entry for this change.

Reply:
Hey, thanks for the review! I thought this PR might be merged first, that's why I comment out for the CI to pass. But up to you, we can also wait for the other one :)

Reply:
if #457 is not that much work we can wait for it to be merged and then merge this. That way we would not have to make an extra PR just to uncomment `mypy` and CI would be happy too. :)

Reply:
@Stannislav On commenting out `mypy` we might have to create an extra PR just to uncomment. So I guess it would be best if we quickly patch up #457 and then we can merge this PR.

Reply:
Or we can have the extra commit and I can squash everything in the end via github interface. It works fine for me too

Reply:
Just submitted a new PR for fixing types in `config_loader`: #462

Reply:
Since there is already a PR for #456, let's just wait and merge that. Then we can merge this and look into the documentation process of type annotations like how to start the whole thing, when should types be necessary for a PR and should they be included in the `Good First Issues` or `First Contribution` since folks who are starting out might not be familiar with it.